tests/utils/utils.js

```diff
@@ -38,6 +39,7 @@ function getAllResourceTypeKeys() {
         return counters.concat(keys);
     });
     // Concatenate each array of resourceType keys into one single array.
+    // eslint-disable-next-line prefer-spread
     return [].concat.apply([], allResourceTypeKeys);
```

Comment:
I think this wants `return [...allResourceTypeKeys];`? Instead of just disabling eslint, but if the intent is to disable all new eslint issues and come back later that is fine.

Reply:
There's apparently more going on here as changing this causes a test to break. I have no idea how the final array is supposed to look.